tests/unit-tests/test-class-sensei-data-cleaner.php

```diff
@@ -432,4 +416,55 @@ public function testSenseiRolesAndCapsDeleted() {
 		$role = get_role( 'teacher' );
 		$this->assertNull( $role, 'Teacher role should be removed overall' );
 	}
+
+	/**
+	 * Ensure the Sensei transients are deleted from the DB.
+	 *
+	 * @covers Sensei_Data_Cleaner::cleanup_all
+	 * @covers Sensei_Data_Cleaner::cleanup_transients
+	 */
+	public function testSenseiTransientsDeleted() {
+		set_transient( 'sensei_123_none_module_lessons', 'value', 0 );
+		set_transient( 'sensei_answers_123_456', 'value', 0 );
+		set_transient( 'sensei_answers_feedback_123_456', 'value', 0 );
+		set_transient( 'quiz_grades_123_456', 'value', 0 );
+		set_transient( 'other_transient', 'value', 0 );
+
+		Sensei_Data_Cleaner::cleanup_all();
+
+		// Flush transients from cache.
+		wp_cache_flush();
+
+		$prefix         = '_transient_';
+		$timeout_prefix = '_transient_timeout_';
+
+		// Ensure the transients and their timeouts were deleted.
+		$this->assertFalse( get_option( "{$prefix}sensei_123_none_module_lessons" ), 'Sensei none_module_lessons transient' );
+		$this->assertFalse( get_option( "{$timeout_prefix}sensei_123_none_module_lessons" ), 'Sensei none_module_lessons transient timeout' );
```

Comment:
In order to do true tests for the transient timeouts, you'll need to set some transients that expire. This is `false` because it was never created in the first place.

Reply:
Whoops. I had expiries set, and for some reason I deleted them just before pushing. "What could possibly go wrong?"

Fixed. I'll wait for CI and then merge.